indigo/distribution.yaml

```diff
@@ -11133,6 +11133,17 @@ repositories:
       url: https://gitlab.com/nasa-jsc-robotics/robodyn_utilities.git
       version: develop
     status: maintained
+  robot_activity:
+    doc:
+      type: git
+      url: https://github.com/snt-robotics/robot_activity.git
+      version: master
+    source:
+      test_pull_requests: true
```

Comment:
I guess the webhook has been created on the new repository as well ?

Reply:
I changed the name of the repo and the webhook is still there, so it should be fine, otherwise I can generate a new one